TASK/IsPedSprinting.md

```diff
@@ -7,9 +7,26 @@ ns: TASK
 // 0x57E457CD2C0FC168 0x4F3E0633
 BOOL IS_PED_SPRINTING(Ped ped);
 ```
-
+Is the ped given sprinting.
 
 ## Parameters
-* **ped**: 
+* **ped**: Ped id.
 
 ## Return value
+Whether or not the ped is sprinting. Note that if the ped is sprinting, it will return `1` instead of `true`.
+
+Code like this:
```

Comment:
Please format these as correct examples, look [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md) for more information. If you'd rather add more description text then you should add that above the Parameters section. This text below return value would be either added to the Return value description or ignored completely.